jira/komand_jira/util/util.py

```diff
@@ -84,7 +84,6 @@ def normalize_user(user, is_cloud = False, logger=logging.getLogger()):
         output["account_id"] = user.accountId
     else:
         output["name"] = user.name
-        output["email_address"] = user.emailAddress
```

Comment:
@r7-mgorny why did we remove this line?

Reply:
It's removed from here but above it is added. `email_address` is returned from api for cloud and on-premise. So I changed it to be added in both places.